flink-ml-lib/src/main/java/org/apache/flink/ml/feature/vectorassembler/VectorAssembler.java

```diff
@@ -133,6 +145,17 @@ public void flatMap(Row value, Collector<Row> out) {
                             : assembleSparse(inputCols, value, vectorSize, nnz);
             out.collect(Row.join(value, Row.of(assembledVec)));
         }
+
+        private void sizeHint(int expectedSize, int currentSize) {
```

Comment:
How about renaming this method to `checkVectorAndNumberSize()`? The current name is inherited from Spark's `VectorSizeHint` operator, so users may find this name hard to understand if they have not known Spark before.